src/ontology/components/livestockbreeds.tsv

```diff
@@ -607,4 +607,31 @@ VBO:0017480	Zemmour (Sheep)		Zemmour	Sheep		240514-128	https://github.com/monarc
+VBO:0017494	OSU 142 (Rainbow trout)		OSU 142	Rainbow trout		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:8022		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				OSU 142	http://purl.obolibrary.org/obo/vbo#most_common_name	https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/1548-8667%281998%29010%3C0075%3ADOLTCL%3E2.0.CO%3B2|https://www.sciencedirect.com/science/article/abs/pii/S0145305X98000391					OSU-142	https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/1548-8667%281998%29010%3C0075%3ADOLTCL%3E2.0.CO%3B2|https://www.sciencedirect.com/science/article/abs/pii/S0145305X98000391																																									https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/1548-8667%281998%29010%3C0075%3ADOLTCL%3E2.0.CO%3B2|https://www.sciencedirect.com/science/article/abs/pii/S0145305X98000391		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648																									
+VBO:0017495	AB (Zebrafish)		AB	Zebrafish		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:7955		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				AB	http://purl.obolibrary.org/obo/vbo#most_common_name		ZDB-GENO-960809-7																																															ZDB-GENO-960809-7	https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648																									
+VBO:0017496	GloFish (Zebrafish)		GloFish	Zebrafish		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:7955		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				GloFish	http://purl.obolibrary.org/obo/vbo#most_common_name	https://shop.glofish.com/collections/glofish-danio					GloFish Danio	https://shop.glofish.com/collections/glofish-danio																																									https://shop.glofish.com/collections/glofish-danio		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648																									
+VBO:0017497	SJD (Zebrafish)		SJD	Zebrafish		240806-181	https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/181	NCBITaxon:7955		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648	VBO:0400000		https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648				SJD	http://purl.obolibrary.org/obo/vbo#most_common_name		ZDB-GENO-990308-9																																															ZDB-GENO-990308-9	https://orcid.org/0000-0003-2826-6444|https://orcid.org/0000-0002-5002-8648																									

```

Comment:
the source should be ZFIN:ZDB-GENO-990308-9
(you forgot the "ZFIN:" prefix)

Reply:
Added